target/linux/ramips/mt76x8/base-files/etc/board.d/01_leds

```diff
@@ -29,6 +29,9 @@ glinet,gl-mt300n-v2)
 	ucidef_set_led_netdev "wifi_led" "wifi" "red:wlan" "wlan0"
 	ucidef_set_led_switch "wan" "wan" "green:wan" "switch0" "0x1"
 	;;
+hak5,wifi-pineapple-mk7)
+	ucidef_set_led_netdev "wifi_led" "wifi" "blue:system" "wlan0"
```

Comment:
Is using this LED for "wifi" vendor behavior? It seems unprobably that the blue LED is used for two different things while the other ones are not used at all.

Reply:
That's correct. The other two LED's are for the user to customize or manage, one they are using the device.

The "three" LEDs are a single RGB LED.

Reply:
Okay. So, since you cannot use a DT trigger then as you have the led-aliases, you will have to define that LED in 01_leds indeed.
However, please use the ucidef_set_led_wlan statement with appropriate trigger here instead, as it will be independent of the interface name. Since we have multiple Wifis, I wonder whether there is a better name than just "wifi_led"?

Reply:
In the factory firmware, wlan0 serves an Open AP. Do you think naming this "open_wifi_led" or "open_ap_led" is better?

Reply:
@Foxtrot Actually, I don't know what would be a good name, as I don't know the device in detail. I just wanted you to think about it, instead of choosing a name by copy-paste. The more important thing here will be to use a phy trigger instead, so we are independent of the interface name.

Reply:
Okay, "open_wifi_led" and using phy0tpt seems right to me, then.